ingest_api/src/collection_publisher.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from pypgstac.db import PgstacDB
+from stac_pydantic import Item
+from src.schemas import DashboardCollection
+from src.utils import (
+    IngestionType,
+    convert_decimals_to_float,
+    get_db_credentials,
+    load_into_pgstac,
+)
+from src.vedaloader import VEDALoader
+
+
+class CollectionPublisher:
+
+    def ingest(self, collection: DashboardCollection):
+        """
+        Takes a collection model,
+        does necessary preprocessing,
+        and loads into the PgSTAC collection table
+        """
+        creds = get_db_credentials(os.environ["DB_SECRET_ARN"])
+        collection = [convert_decimals_to_float(collection.dict(by_alias=True))]
```

Comment:
```suggestion
        collection = [convert_decimals_to_float(collection.dict(by_alias=True, exclude_none=True))]
```

This will prevent optional-but-not-nullable fields, e.g. `assets` on a `Collection`.